### DISCUSSION TITLE
Noob question - Standalone/portable mode

### DISCUSSION THREAD

**Discussion by Conflituoso** [Alex313031/thorium-legacy#19] [Q&A]
Hi.
To use standalone/portable mode, should I always start with THORIUM.BAT?
Or should I run THORIUM.BAT just once and then use thorium.exe?

**Comment by llExSt** [comment]
The [README.txt](https://github.com/Alex313031/thorium-win7/blob/main/portable/README.win) inside the zip explains it. Basically directly executing the .exe will create the profile inside your AppData, wheres executing the .bat will create the profile in the supplied USER_DATA folder.
I also notice it leaves behind registry entries after executing the .bat file. I think for the sake of "portable" sense it shouldn't or deletes them itself after closing..

**Comment by Alex313031** [comment]
@Conflituoso @llExSt I'll look into making Thorium not write to the registry in portable mode.
